WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -221,15 +221,15 @@ public void onCreate() {
         mContext = this;
         long startDate = SystemClock.elapsedRealtime();
 
-        // Init WellSql
+        // Initialize WellSql
```

Comment:
These `"Init"` were supposed to _quickly_ describe what the code blocks do. The change to `"Initialize"` only made these comments _take longer_ to read. The word `"Init"` is understandable enough to everyone. 

Can we change them back? 🙂